backend/stakepoold/stakepool/stakepool.go

```diff
@@ -333,13 +341,13 @@ func (spd *Stakepoold) AddMissingTicket(ticketHash []byte) error {
 		return err
 	}
 
-	tx, err := spd.WalletConnection.RPCClient().GetRawTransaction(hash)
+	tx, err := spd.NodeConnection.GetRawTransaction(hash)
```

Comment:
As far as I can tell this has been a dcrd rpc from the beginning...

Reply:
dcrd's command
https://github.com/decred/dcrd/blob/ee4a0e2e2a0e7dc2e71ae807e7586c5fe03ed407/rpc/jsonrpc/types/chainsvrcmds.go#L682-L689

dcrwallets's
https://github.com/decred/dcrwallet/blob/fe6e4c8c0cce1065111cacd5b11193595b7e386f/rpc/jsonrpc/types/methods.go#L489-L493

Reply:
So, perhaps this was what we wanted, but the naming was just wrong, let me look into what it's used for a bit more.

Reply:
It's used for the wallet rpc "addticket", which seems to accept a raw has based on the variable name here, but I'm unable to find a comment that confirms this.

https://github.com/decred/dcrd/blob/eb5bb6d4192acbf57dfedcf57aa6654adae10a6d/rpcclient/wallet.go#L2325-L2341

Reply:
I order to use getrawtransaction on dcrd "The transaction index must be enabled to query the blockchain (specify --txindex)"

Reply:
for transaction:
cc9d15715a51a0981a6a1911863122279fb05a1b2ff12f067762fc3317e85300

dcrwallet's hex:
0100000002380b3eadc0e9ac8446e350d374fb33e626e808f1f57d0f3f58adf2d7c663c1af0000000000ffffffff380b3eadc0e9ac8446e350d374fb33e626e808f1f57d0f3f58adf2d7c663c1af0100000000ffffffff05c3224c8c01000000000018baa914515d6c01df8f3d908da72da81204b51b26a864da8700000000000000000000206a1edc29731dd5919af414329223d7f64448c17efa95dcd60300000000000058000000000000000000001abd76a914000000000000000000000000000000000000000088ac00000000000000000000206a1e79d29020d3a558bf08ed3a7512f5ff6c992b77561361488c010000000058000000000000000000001abd76a914000000000000000000000000000000000000000088ac000000000000000002dcd6030000000000**00000000ffffffff**6b483045022100874e26f9ef3a28d5f0e1e72da61fca07cb35df019507bb124c7a33389cf8a55d02205c87d0746730bbb0f8b0e1d9b8e703bfbe4b69adf5dcd32cfa303ddfe31e2adc0121039ff5a6a373eb15d9b71505e78a9d39750b1e9616bc4af7fef82ee32cce9dc20b1361488c01000000**00000000ffffffff**6a47304402206f6a934e4206730d0d43c78ffcea9ca05c56846f751baac34e9e256d1558d1fb0220413a9ab43446c39bdf37fa7ddf300f5f7bbd138fd4d4ef60ff3f6f0be2e8910b0121039ff5a6a373eb15d9b71505e78a9d39750b1e9616bc4af7fef82ee32cce9dc20b

dcrd's hex:
0100000002380b3eadc0e9ac8446e350d374fb33e626e808f1f57d0f3f58adf2d7c663c1af0000000000ffffffff380b3eadc0e9ac8446e350d374fb33e626e808f1f57d0f3f58adf2d7c663c1af0100000000ffffffff05c3224c8c01000000000018baa914515d6c01df8f3d908da72da81204b51b26a864da8700000000000000000000206a1edc29731dd5919af414329223d7f64448c17efa95dcd60300000000000058000000000000000000001abd76a914000000000000000000000000000000000000000088ac00000000000000000000206a1e79d29020d3a558bf08ed3a7512f5ff6c992b77561361488c010000000058000000000000000000001abd76a914000000000000000000000000000000000000000088ac000000000000000002dcd6030000000000**de62030008000000**6b483045022100874e26f9ef3a28d5f0e1e72da61fca07cb35df019507bb124c7a33389cf8a55d02205c87d0746730bbb0f8b0e1d9b8e703bfbe4b69adf5dcd32cfa303ddfe31e2adc0121039ff5a6a373eb15d9b71505e78a9d39750b1e9616bc4af7fef82ee32cce9dc20b1361488c01000000**de62030008000000**6a47304402206f6a934e4206730d0d43c78ffcea9ca05c56846f751baac34e9e256d1558d1fb0220413a9ab43446c39bdf37fa7ddf300f5f7bbd138fd4d4ef60ff3f6f0be2e8910b0121039ff5a6a373eb15d9b71505e78a9d39750b1e9616bc4af7fef82ee32cce9dc20b

You can see only a slight difference in the output hexes...

Reply:
after quite a bit of sleuthing

<details close>
<summary> breakdown</summary>

0100000002380b3eadc0e9ac8446e350d374fb33e626e808f1f57d0f3f58adf2d7c663c1af0000000000ffffffff380b3eadc0e9ac8446e350d374fb33e626e808f1f57d0f3f58adf2d7c663c1af0100000000ffffffff05c3224c8c01000000000018

vout 0 [baa914515d6c01df8f3d908da72da81204b51b26a864da87]

0000000000000000000020

vout 1 [6a1edc29731dd5919af414329223d7f64448c17efa95dcd60300000000000058]

000000000000000000001a

vout 2 [bd76a914000000000000000000000000000000000000000088ac]

0000000000000000000020

vout 3 [6a1e79d29020d3a558bf08ed3a7512f5ff6c992b77561361488c010000000058]

000000000000000000001a

vout 4 [bd76a914000000000000000000000000000000000000000088ac]

000000000000000002

dcd6030000000000 

block height [de620300] 

block index [08000000] 

6b

vin 0 [483045022100874e26f9ef3a28d5f0e1e72da61fca07cb35df019507bb124c7a33389cf8a55d02205c87d0746730bbb0f8b0e1d9b8e703bfbe4b69adf5dcd32cfa303ddfe31e2adc0121039ff5a6a373eb15d9b71505e78a9d39750b1e9616bc4af7fef82ee32cce9dc20b]

1361488c01000000de62030008000000 6a

vin 1 [47304402206f6a934e4206730d0d43c78ffcea9ca05c56846f751baac34e9e256d1558d1fb0220413a9ab43446c39bdf37fa7ddf300f5f7bbd138fd4d4ef60ff3f6f0be2e8910b0121039ff5a6a373eb15d9b71505e78a9d39750b1e9616bc4af7fef82ee32cce9dc20b]


</details>

the difference translates into a null block height and index for dcrwallet.

Not sure if the height and index are required yet.

Reply:
Another difference is that the wallet only knows about tx that concern it.

Reply:
So, thinking about the last reason, I guess this does need to be a dcrd command, and --txindex must be specified.  If I understand the purpose for this command correctly, the wallet must not have seen that txid, so we need to ask dcrd...  I think this change is correct.